### DISCUSSION TITLE
Custom variants is not reflected. checkbox.ts

### DISCUSSION THREAD

**Discussion by nbeppu** [heroui-inc/heroui#1345] [Feedback]
If `boder-2` is specified in the following places, the boderWidth of custom variants is not reflected.
Shouldn't this be `border-medium`?

nextui/packates/core/theme/src/components/chexbox.ts

```
   "flex-shrink-0",
      "overflow-hidden",
      // before
      "before:content-['']",
      "before:absolute",
      "before:inset-0",
      "before:border-solid",
      "before:border-2",
      "before:box-border",
      "before:border-default",
      // after
      "after:content-['']",
```

**Comment by jrgarciadev** [comment]
Hey @nbeppu yes, we're still migrating all tailwind-based values to NextUI units/layout

**Reply by nbeppu** [reply]
Thanks. I will wait for the update.
